jsapp/js/projects/projectViews/projectsFilterEditor.tsx

```diff
@@ -123,19 +130,40 @@ export default function ProjectsFilterEditor(props: ProjectsFilterEditorProps) {
       {/* Filter value */}
       <div className={styles.column}>
         {!props.hideLabels && (
-          <span className={styles.label}>{t('Value')}</span>
+          <span className={styles.label}>
+            {isCountryFilterSelected() ? t('Select country') : t('Value')}
+          </span>
```

Comment:
Let's leave it as it was before. If you add multiple filters you will see what I mean :) These labels are working as a "column headers" kinda. I decided to do it this way, so it's possible to use a filter editor with labels as singular instance.